### PR TITLE
[All] updating validation tool info

### PR DESCRIPTION
**Pull request by davidchesnut**
Reworked content in article for validation. Now it only mentions how to use office-toolbox package for validation. Both for yo office, and for custom projects.

Some other minor cleanup work was done as well.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[ea2e74a](https://github.com/OfficeDev/office-js-docs-pr/pull/1149/commits/ea2e74aa13f0c03e4f346a59a117a9afd2ab7741)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/troubleshoot-manifest.md](https://github.com/OfficeDev/office-js-docs-pr/blob/davech-validate/docs/testing/troubleshoot-manifest.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C1%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201907012357248968-1149%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=GKPvXRuVZ40oQjMz7%2B5TZZwFDcIvbMbNA%2BERyIjuhaE%3D&st=2019-07-01T23%3A55%3A33Z&se=2019-08-02T00%3A00%3A33Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[000c844](https://github.com/OfficeDev/office-js-docs-pr/pull/1149/commits/000c844d60d373fadf32247cbf5b7fdf090d879c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/troubleshoot-manifest.md](https://github.com/OfficeDev/office-js-docs-pr/blob/davech-validate/docs/testing/troubleshoot-manifest.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C2%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201907020002244997-1149%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=cSFbwAcBPVsNBlQc%2B3dD5jzGdseLlTQ2qsfN4xdACXY%3D&st=2019-07-01T23%3A58%3A21Z&se=2019-08-02T00%3A03%3A21Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by akrantz** [comment]
It looks much better.

File: docs/testing/troubleshoot-manifest.md [hunk 1]
```diff
@@ -1,73 +1,54 @@
...
-	```command&nbsp;line
-	npm install -g office-addin-validator
-	```
-	
-	> [!NOTE]
-	> If you already have Yo Office installed, upgrade to the latest version, and the validator will be installed as a dependency.
+1. Install [Node.js](https://nodejs.org/download/).
 
-3. Run the following command to validate your manifest. Replace MANIFEST.XML with the path to the manifest XML file.
+2. Open a command prompt / terminal as an administrator, and run the Office Add-in validator on your manifest file. The Office Add-in validator is in the office-toolbox package.
```
Comment (kbrandl) [inline docs/testing/troubleshoot-manifest.md]:
```suggestion
2. Run the following command in the root directory of your project. Replace `MANIFEST_FILE` with the name of the manifest file.
```

File: docs/testing/troubleshoot-manifest.md [hunk 2]
```diff
@@ -1,73 +1,54 @@
...
 
 	```command&nbsp;line
-	validate-office-addin MANIFEST.XML
+	npx office-toolbox validate -m manifest.xml
```
Comment (kbrandl) [inline docs/testing/troubleshoot-manifest.md]:
```suggestion
	npx office-toolbox validate -m MANIFEST_FILE
```

File: docs/testing/troubleshoot-manifest.md [hunk 3]
```diff
@@ -1,73 +1,54 @@
...
 	```
 
 ## Validate your manifest against the XML schema
 
-To help ensure that the manifest file follows the correct schema, including any namespaces for the elements you are using. If you copied elements from other sample manifests double check you also **include the appropriate namespaces**. You can validate a manifest against the [XML Schema Definition (XSD)](https://github.com/OfficeDev/office-js-docs-pr/tree/master/docs/overview/schemas) files. You can use an XML schema validation tool to perform this validation. 
-
-
+To help ensure that the manifest file follows the correct schema, including any namespaces for the elements you are using. If you copied elements from other sample manifests double check you also **include the appropriate namespaces**. You can validate a manifest against the [XML Schema Definition (XSD)](https://github.com/OfficeDev/office-js-docs-pr/tree/master/docs/overview/schemas) files. You can use an XML schema validation tool to perform this validation.
```
Comment (kbrandl) [inline docs/testing/troubleshoot-manifest.md]: I can't spot what you changed in this paragraph, but the opening sentence here seems malformed/incomplete. Do you want to fix that as part of this PR?

**Comment by o365devx** [comment]
Docs Build status updates of commit _[84beca9](https://github.com/OfficeDev/office-js-docs-pr/pull/1149/commits/84beca9b90d5b53a6b174f151e3a0b681daa6e56)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/troubleshoot-manifest.md](https://github.com/OfficeDev/office-js-docs-pr/blob/davech-validate/docs/testing/troubleshoot-manifest.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C5%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201907051923495250-1149%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=s4RDKJwPpmeG6hYwjjTeZNKAdtq4FOmzsv9QJBwGtr0%3D&st=2019-07-05T19%3A19%3A53Z&se=2019-08-05T19%3A24%3A53Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/testing/troubleshoot-manifest.md (hunk 3 above)
Comment (davidchesnut) [reply docs/testing/troubleshoot-manifest.md]: @kbrandl I put in a rewrite for this. See if this works!

**Comment by o365devx** [comment]
Docs Build status updates of commit _[d61fbe0](https://github.com/OfficeDev/office-js-docs-pr/pull/1149/commits/d61fbe03f345b253e16cf30283d6fea8c3168e2d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/troubleshoot-manifest.md](https://github.com/OfficeDev/office-js-docs-pr/blob/davech-validate/docs/testing/troubleshoot-manifest.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C5%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201907051933525054-1149%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=cBYq0JYr9aC4HMebar1exIUhzGNPQjpH6JcdaSuFRE0%3D&st=2019-07-05T19%3A29%3A56Z&se=2019-08-05T19%3A34%3A56Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/testing/troubleshoot-manifest.md [hunk 4]
```diff
@@ -1,73 +1,54 @@
...
-	```command&nbsp;line
-	npm install -g office-addin-validator
-	```
-	
-	> [!NOTE]
-	> If you already have Yo Office installed, upgrade to the latest version, and the validator will be installed as a dependency.
+1. Install [Node.js](https://nodejs.org/download/).
 
-3. Run the following command to validate your manifest. Replace MANIFEST.XML with the path to the manifest XML file.
+2. Open a command prompt / terminal as an administrator, and run the Office Add-in validator on your manifest file. The Office Add-in validator is in the office-toolbox package.
```
[the remark this answers is not in the document]
Comment (davidchesnut) [reply docs/testing/troubleshoot-manifest.md]: Thanks @akrantz. Fixed!

**Comment by davidchesnut** [comment]
@kbrandl accepted all your suggestions. Bit of a rewrite on the intro sentence to break it into smaller sentences so see if that makes sense. Thanks!

**Pull request merged**